apps/crn-frontend/src/network/teams/__tests__/TeamProfile.test.tsx

```diff
@@ -1,24 +1,31 @@
-import { Suspense } from 'react';
-import { RecoilRoot } from 'recoil';
-import { render, waitFor } from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
-import { MemoryRouter, Route } from 'react-router-dom';
-import { createTeamResponse } from '@asap-hub/fixtures';
-import { network } from '@asap-hub/routing';
 import {
   Auth0Provider,
   WhenReady,
 } from '@asap-hub/crn-frontend/src/auth/test-utils';
-
+import {
+  createListEventResponse,
+  createTeamResponse,
+} from '@asap-hub/fixtures';
+import { network } from '@asap-hub/routing';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Suspense } from 'react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import { RecoilRoot } from 'recoil';
+import { getEventsFromAlgolia } from '../../../events/api';
+import { getResearchOutputs } from '../../../shared-research/api';
 import { createResearchOutputListAlgoliaResponse } from '../../../__fixtures__/algolia';
-import TeamProfile from '../TeamProfile';
 import { getTeam } from '../api';
 import { refreshTeamState } from '../state';
-import { getResearchOutputs } from '../../../shared-research/api';
+import TeamProfile from '../TeamProfile';
 
 jest.mock('../api');
 jest.mock('../groups/api');
 jest.mock('../../../shared-research/api');
+jest.mock('../../../events/api');
+const mockEvents = getEventsFromAlgolia as jest.MockedFunction<
```

Comment:
```suggestion
const mockGetEventsFromAlgolia = getEventsFromAlgolia as jest.MockedFunction<
```

We always do this as a convention